kytos/core/napps/manager.py

```diff
@@ -95,6 +95,11 @@ def install(self, napp_uri, enable=True):
 
         log.info("New NApp installed: %s", napp)
 
+        napp = NApp.create_from_json(dst/'kytos.json')
+        log.info('Installing Napp Dependencies:')
```

Comment:
I  think that @cemsbr suggested to put the "log.info" about "installing dependencies" inside an if clause, since that if there are no dependencies there is no reason to log this message.